cmd/commands/permissions.go

```diff
@@ -64,3 +87,54 @@ func resetPermissionsCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+type permissionsExportWriter struct{}
+
+func (pew permissionsExportWriter) Write(p []byte) (int, error) {
+	fmt.Printf("%v", string(p))
+	return len(p), nil
+}
+
+func exportPermissionsCmdF(command *cobra.Command, args []string) error {
+	a, err := cmd.InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if license := a.License(); license == nil {
+		return errors.New(utils.T("cli.license.critical"))
+	}
+
+	writer := permissionsExportWriter{}
```

Comment:
I meant here, make more sense to use directly os.Stdout, instead your own writter to print to the stdout

Reply:
Great call. Works perfectly.